CONTRIBUTING.md

```diff
@@ -147,6 +147,30 @@ You can run the end-to-end tests automatically on your forked project by followi
 4. In the run-e2e-tests workflow page, select "Run workflow" and run it on the desired branch.
 
 > :warning: **Don't automatically run end-to-end tests on branch push or PRs**. A malicious attacker can submit a pull request to attack your AWS account. Ideally, use a blank account without any important workload/data, and limit `AWS_ROLE_ARN_TO_ASSUME` permission to least minimum privilege.
+
+
+### Examples
+
+As part of the repo you will find an examples folder at the root. This folder contains examples (written with CDK for now) of deployable AWS Lambda functions using Powertools.
+
+To test your updates with these examples you just have to:
+
+1. Build your local version of *aws-lambda-powertools-typescript* npm packages with `npm run lerna-package`
+1. Update their references in examples
+    ```
+    cd examples/cdk
+    npm install ../../packages/**/dist/aws-lambda-powertools-*
```

Comment:
Same as above.

Reply:
hmm weird . it worked on mine. will double check

Reply:
My bad, I didn't rerun npm run lerna-ci and a package was missing.